packages/portal/swagger-ui-web-component/src/element.js

```diff
@@ -1,4 +1,5 @@
 import SwaggerUI from 'swagger-ui'
+import swaggerUIStyles from 'swagger-ui/dist/swagger-ui.css'
```

Comment:
Doesn't importing the styles defeat the purpose? @davidma415 check me here

Reply:
Unsure what the best approach is, but yeah I don't think importing the styles fits our use case. Especially in the case that we currently allow theming (e.g. in the dev portal) and it would also affect those colors.

<img width="1457" alt="Screenshot 2024-03-04 at 7 51 39 AM" src="https://github.com/Kong/public-ui-components/assets/40131297/c867a64a-97ef-43b0-a394-a6bdf2c807e9">

Reply:
Per the team standup discussion, we're now overriding the css background on operations to match our original spec renderer.
We still do need to import the swagger-ui stylesheet for basic styles.

Reply:
<img width="1100" alt="Screenshot 2024-03-04 at 10 31 23 AM" src="https://github.com/Kong/public-ui-components/assets/2568272/055326ea-a6b3-4b63-8240-9676a5a7eaf0">